lab-brianna/index.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+const product = require('./lib/call-apply.js');
+console.log(product);
+
+let packet = new product.MegaCookie('Chocolate Chip', 'Large');
+console.log(packet);
+
+product.places.addToCountries('Canada','UK', 'France');
```

Comment:
**TODO FOR LOST POINTS**
Code broke at Line 9 when I ran ``npm start``. You don't have a ``product.places.addToCountries`` function.